### ISSUE TITLE
Bot occassionally crashes when a voice command is executed

### ISSUE THREAD

**Issue by TTtie** [TTtie/Pyro-Bot#1]
Happens rarely but it's annoying.

**Comment by TTtie** [comment]
Seems to be possibly invalid?

**Comment by TTtie** [comment]
Stacktrace: 
```
events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: Invalid channel ID: null
    at Client.getChannel (Pyro-Bot\node_modules\eris\lib\Client.js:312:19)
    at Client.leaveVoiceChannel (Pyro-Bot\node_modules\eris\lib\Client.js:255:28)
    at Shard.wsEvent (Pyro-Bot\node_modules\eris\lib\gateway\Shard.js:687:33)
    at WebSocket.ws.onmessage (Pyro-Bot\node_modules\eris\lib\gateway\Shard.js:1462:34)
    at WebSocket.onMessage (Pyro-Bot\node_modules\eris\node_modules\ws\lib\EventTarget.js:103:16)
    at emitTwo (events.js:125:13)
    at WebSocket.emit (events.js:213:7)
    at Receiver._receiver.onmessage (Pyro-Bot\node_modules\eris\node_modules\ws\lib\WebSocket.js:146:54)
    at Receiver.dataMessage (Pyro-Bot\node_modules\eris\node_modules\ws\lib\Receiver.js:389:14)
    at Receiver.getData (Pyro-Bot\node_modules\eris\node_modules\ws\lib\Receiver.js:330:12)
    at Receiver.startLoop (Pyro-Bot\node_modules\eris\node_modules\ws\lib\Receiver.js:165:16)
    at Receiver.add (Pyro-Bot\node_modules\eris\node_modules\ws\lib\Receiver.js:139:10)
    at TLSSocket._ultron.on (Pyro-Bot\node_modules\eris\node_modules\ws\lib\WebSocket.js:142:22)
```

**Comment by TTtie** [comment]
Fixed with bcc0db3.  
The reason this was erroring is becuase some people ran the command outside a guild, I think.
